lib/vquic/ngtcp2.c

```diff
@@ -1569,8 +1701,14 @@ static CURLcode ng_has_connected(struct Curl_easy *data,
     if(result)
       return result;
     infof(data, "Verified certificate just fine");
-#else
+#elif defined(USE_GNUTLS)
     result = Curl_gtls_verifyserver(data, conn, conn->quic->ssl, FIRSTSOCKET);
+#elif defined(USE_WOLFSSL)
+    char *snihost = Curl_ssl_snihost(data, SSL_HOST_NAME(), NULL);
+    if(!snihost ||
```

Comment:
Should this really be an *OR*? The latter expression makes less sense then.

Reply:
I took this from `lib/vtls/wolfssl.c_616` as I thought QUIC and TLS should behave the same in this case. Have I mistunderstood?

Reply:
No, I believe it is correct!